src/styles/carousel.scss

```diff
@@ -201,3 +205,18 @@ $button: #{$fd-namespace}-button;
     }
   }
 }
+
+@media (max-width: 21.25rem) {
```

Comment:
The `21.25rem` sounds like a magical value, fiori3 specs say nothing about this breakpoint, the smallest one is `< 600px`. 

There is nothing about changing paddings or anything at some breakpoint in wiki https://wiki.wdf.sap.corp/wiki/pages/viewpage.action?pageId=2059230114. 

The container is `15.5rem` in width what is `248px` and it's okay for the `320px` screen and there is no sense to support smaller devices. 

Instead, I propose at `< 600px` breakpoint hide the items' dots and only leave prev/next buttons, it's also not covered in fiori3 specs but i think it will be more suitable approach, @InnaAtanasova @droshev what do you think?

Reply:
we can't propose or come up with media queries if they are not specified in Fiori :)
Here's information about what should be hidden/visible on mobile: https://experience.sap.com/internal/fiori-design-web/carousel/#responsiveness